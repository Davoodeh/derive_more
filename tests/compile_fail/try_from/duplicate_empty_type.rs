#[derive(derive_more::TryFrom)]
#[try_from(usize)]
enum Enum {
    Field,
    Field2,
}

fn main() {}
