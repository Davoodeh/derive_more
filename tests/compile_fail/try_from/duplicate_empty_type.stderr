error: empty variant not supported when using non-repr `try_from`
 --> tests/compile_fail/try_from/duplicate_empty_type.rs:4:5
  |
4 |     Field,
  |     ^^^^^
